docs/docsite/rst/dev_guide/testing/sanity/import.rst

```diff
@@ -1,5 +1,39 @@
 import
 ======
 
-All Python imports in ``lib/ansible/modules/`` and ``lib/ansible/module_utils/`` which are not from the Python standard library
-must be imported in a try/except ImportError block.
+All Python imports in the directories specified that are not from a limited set of possibilities listed below
+must be imported in a try/except ImportError block as follows:
+
+.. code-block:: python
+
+    # Instead of 'import another_library', do:
+
+    try:
+        import another_library
+        HAS_ANOTHER_LIBRARY = True
```

Comment:
For readability, it's probably better to have this in the `else:` block.

Reply:
Fixed in 4f9e1277be264add28921b348cdaf820b81a2a9f.